.gitignore

```diff
@@ -43,3 +43,6 @@ microsoft-teams-library-js.sln
 artifacts/
 bin/
 obj/
+
+#VS Code Local Files
+*.sln
```

Comment:
For some reason, VS code has started to create soln files that git then wants to check in. I don't want sln files checked in, and therefore I have added them to our .gitignore file.

Reply:
I don't really care that much about this but _normally_ sln files would be checked in so that everyone on the team builds from the same solution. On our team I think this only matters for the blazor app so it's probably not terribly important.